src/Nexus.UI/Components/AttachmentView.razor

```diff
@@ -110,14 +113,23 @@
         _attachments = default;
         StateHasChanged();
 
-        try
-        {
-            await Client.Catalogs.DeleteAttachmentAsync(Catalog.Id, attachment);
-            await GetAttachmentsAsync();
-        }
-        catch (Exception ex)
+        bool? result = await DialogService.ShowMessageBox(
+            "Warning",
+            "Do you really want to delete the selected attachment?",
+            yesText:"Delete!", cancelText:"Cancel");
+        bool deleteConfirmed = result == null ? false : true;
```

Comment:
Dies this mean that if result == false, then result = true?

Reply:
I just updated the previous comment, it had a small logic error

Reply:
Okay I see. `result == false` will only ever happen when `noText` parameter is given.